tla-types/src/main/scala/at/forsyte/apalache/tla/types/TlaType1TagPrinter.scala

```diff
@@ -0,0 +1,20 @@
+package at.forsyte.apalache.tla.types
+
+import at.forsyte.apalache.tla.lir.{TypeTag, Typed, Untyped}
+import at.forsyte.apalache.tla.lir.io.TypeTagPrinter
+import at.forsyte.apalache.tla.typecheck.TlaType1
+
+class TlaType1TagPrinter extends TypeTagPrinter {
+  def apply(tag: TypeTag): String = {
+    tag match {
+      case Untyped()           => "Untyped"
+      case Typed(tt: TlaType1) => tt.toString
+      // unexpected type, output as much as we can
+      case Typed(_) => "Typed[%s](%s)".format(tag.getClass.getSimpleName, tag)
```

Comment:
Missing `case somethingElse` ? Make it throw explicitly (or write something along the lines of `UNDEFINED`), so we don't get surprised by crashes on `TlaType7` tags in the future.

Reply:
No missing case, as `Typed` and `Untyped` are the only case classes. `TypeTag` is sealed.

Reply:
Yeah, for whatever reason my brain just ignored l.13